docs/configuration/plugins.md

```diff
@@ -2,7 +2,7 @@
 
 ## PLUGINS
 
-Default: Empty
+Default: `Empty`
```

Comment:
"Empty" isn't a literal value. We should either leave this as is, or change it to "`[]` (empty)"

Reply:
@jeremystretch thanks. Will switch it to []